app/index/template.hbs

```diff
@@ -1,9 +1,9 @@
 <div class="index-content">
   <section class="latest-contribution">
     {{#each (take 1 (sort-by 'insertedAt:desc' model.contributions))
-      as |latestContribution|}}
+      as |latestContribution|}}<a href="/day/{{latestContribution.date}}">
```

Comment:
In an ember app, you shouldn't use plain old html5 links, these result in a hardreload of the entire page.

Use
```handlebars
{{#link-to 'day' latestContribution.date}}<!-- link content -->{{/link-to}}
```
instead :)